Test's don't build with CUDA 9.1 (Visual Studio 2015)

When building on my laptop (Visual Studio 2015, CUDA 9.1), most (all?) test files fail to build due to this compilation error.
```
C:\Users\Robadob\fgpu2\build\tests\googletest-src\googletest\include\gtest/internal/gtest-internal.h(1157): error : no instance of function template "testing::internal::ElemFromListImpl<testing::internal::IndexSequence<I...>>::Apply [with I=<0ULL>]" matches the argument list
1>C:\Users\Robadob\fgpu2\build\tests\googletest-src\googletest\include\gtest/internal/gtest-internal.h(1157): error : no instance of function template "testing::internal::ElemFromListImpl<testing::internal::IndexSequence<I...>>::Apply [with I=<0ULL>]" matches the argument list
1>C:\Users\Robadob\fgpu2\build\tests\googletest-src\googletest\include\gtest/internal/gtest-internal.h(1157): error : no instance of function template "testing::internal::ElemFromListImpl<testing::internal::IndexSequence<I...>>::Apply [with I=<0ULL>]" matches the argument list
1>C:\Users\Robadob\fgpu2\build\tests\googletest-src\googletest\include\gtest/internal/gtest-internal.h(1157): error : no instance of function template "testing::internal::ElemFromListImpl<testing::internal::IndexSequence<I...>>::Apply [with I=<0ULL>]" matches the argument list
1>C:\Users\Robadob\fgpu2\build\tests\googletest-src\googletest\include\gtest/internal/gtest-internal.h(1157): error : no instance of function template "testing::internal::ElemFromListImpl<testing::internal::IndexSequence<I...>>::Apply [with I=<0ULL>]" matches the argument list
1>  test_host_random.cu
1>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V140\BuildCustomizations\CUDA 9.1.targets(707,9): error MSB3721: The command ""C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\bin\nvcc.exe" -gencode=arch=compute_52,code=\"sm_52,compute_52\" -gencode=arch=compute_52,code=\"compute_52,compute_52\" --use-local-env --cl-version 2015 -ccbin "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\bin\x86_amd64" -x cu -rdc=true -IC:\Users\Robadob\fgpu2\include -IC:\Users\Robadob\fgpu2\tests -IC:\Users\Robadob\fgpu2\externals -I\..\include -I"C:\Users\Robadob\fgpu2\build\tests\googletest-src\googletest\include" -I"C:\Users\Robadob\fgpu2\build\tests\googletest-src\googletest" -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include"  -G   --keep-dir x64\Debug -maxrregcount=0  --machine 64 --compile -cudart static --expt-relaxed-constexpr --Wreorder -Xcompiler="/EHsc /wd4505 -Zi -Ob0" -g   -D_WINDOWS -D_DEBUG -DDEBUG -D"CMAKE_INTDIR=\"Debug\"" -DWIN32 -D_WINDOWS -D"CMAKE_INTDIR=\"Debug\"" -D_MBCS -Xcompiler "/EHsc /W4 /nologo /Od /FS /Zi /RTC1 /MDd /GR" -o tests.dir\Debug\test_host_random.obj "C:\Users\Robadob\fgpu2\tests\test_cases\runtime\test_host_random.cu"" exited with code 1.
1>  Compiling CUDA source file ..\..\tests\test_cases\runtime\host_reduction\test_histogram_even.cu...
```

--- a/README.md
+++ b/README.md
@@ -119,6 +119,8 @@ The first time CMake is configured with `-DBUILD_TESTS=ON` an internet connectio
 Subsequent reconfigures will attempt to update this copy, but will continue if updating fails.
 Automatic updating of GoogleTest can be disabled by passing `-DBUILD_TESTS=OFF`.
 
+*Known Issues:* The tests do not build under the combination of CUDA 9.1 and Visual Studio 2015. Use CUDA 10.1 or newer if you require the tests.
+
 ##### Documentation
 
 If you wish to build the documentation, [doxygen](http://www.doxygen.nl/) must be available on your system.

--- a/README.md
+++ b/README.md
@@ -119,7 +119,7 @@ The first time CMake is configured with `-DBUILD_TESTS=ON` an internet connectio
 Subsequent reconfigures will attempt to update this copy, but will continue if updating fails.
 Automatic updating of GoogleTest can be disabled by passing `-DBUILD_TESTS=OFF`.
 
-*Known Issues:* The tests do not build under the combination of CUDA 9.1 and Visual Studio 2015. Use CUDA 10.1 or newer if you require the tests.
+*Known Issues:* The tests do not build under the combination of Visual Studio 2015 and CUDA 9.0 or 9.1. Use CUDA 9.2 or newer if you require the tests.
 
 ##### Documentation
 

